### ISSUE TITLE
NAGtools not installed

### ISSUE THREAD

**Issue by d-torrance** [Macaulay2/M2#1177]
The following test in `NumericalAlgebraicGeometry` requires the `NAGtools` package:

https://github.com/Macaulay2/M2/blob/ea039800e10a6d8e4708312ebb5edac3fda88479/M2/Macaulay2/packages/NumericalAlgebraicGeometry/extraNAGtypes.m2#L287-L295

However, `NAGtools` is not included in `=distributed-packages`, and so it's not installed.  This results in a failing test:

```
i4 : check_2 NumericalAlgebraicGeometry
--running test 2 of package NumericalAlgebraicGeometry on line 296 in file /usr/share/Macaulay2/./NumericalAlgebraicGeometry/extraNAGtypes.m2
--    rerun with: check_2 "NumericalAlgebraicGeometry"
--making test results in file /tmp/M2-7696-0/3.out
ulimit -c unlimited; ulimit -t 700; ulimit -m 850000; ulimit -v 850000; ulimit -s 8192; ulimit -n 512; cd /tmp/M2-7696-0/4-rundir/; GC_MAXIMUM_HEAP_SIZE=400M /usr/bin/M2-binary --silent --print-width 77 --stop --int --no-readline --no-randomize -e 'needsPackage("NumericalAlgebraicGeometry", Reload => true, FileName => "/usr/share/Macaulay2/NumericalAlgebraicGeometry.m2")' <"/tmp/M2-7696-0/3.m2" >>"/tmp/M2-7696-0/3.tmp" 2>&1
/tmp/M2-7696-0/3.tmp:0:1: (output file) error: Macaulay2 exited with status code 1
/tmp/M2-7696-0/3.m2:0:1: (input file)
M2: *** Error 1
stdio:4:18:(3): error: error occurred running test for package NumericalAlgebraicGeometry: 2
```

```
profzoom@gloria:~$ cat /tmp/M2-7696-0/3.tmp
-- -*- M2-comint -*- hash: 2104419036
--loading configuration for package "NumericalAlgebraicGeometry" from file /home/profzoom/.Macaulay2/init-NumericalAlgebraicGeometry.m2
--loading configuration for package "PHCpack" from file /home/profzoom/.Macaulay2/init-PHCpack.m2
--loading configuration for package "Bertini" from file /home/profzoom/.Macaulay2/init-Bertini.m2

i1 : --/usr/share/Macaulay2/./NumericalAlgebraicGeometry/extraNAGtypes.m2:296: location of test code
     
     needsPackage "NAGtools"
stdio:3:1:(3): error: file not found on path: "NAGtools.m2"
```

Should `NAGtools` be installed, or should this test be removed?

**Comment by antonleykin** [comment]
I have removed the test in https://github.com/Macaulay2/M2/pull/1096/commits/718c3dddf528ce4d2b6afdb5847c5b32a875a562

We should aim to remove `NAGtools.m2` entirely. Here are all places where it is mentioned:
```
M2/Macaulay2/packages/NAGtools.m2:6:     "NAGtools",
M2/Macaulay2/packages/NAGtools.m2:200:endPackage "NAGtools" 
M2/Macaulay2/packages/NumericalAlgebraicGeometry/SYSTEMS/EXA/Vision/degreeBifocal.m2:2:needsPackage "NAGtools"
M2/Macaulay2/packages/NumericalAlgebraicGeometry/SYSTEMS/monodromy/degree-computation.m2:1:needsPackage "NAGtools"
M2/Macaulay2/packages/NumericalAlgebraicGeometry/SYSTEMS/monodromy/sparse-system-via-polynomials.m2:2:needsPackage "NAGtools"
M2/Macaulay2/packages/NumericalAlgebraicGeometry/SYSTEMS/monodromy/sparse-system.m2:2:needsPackage "NAGtools"
M2/Macaulay2/packages/NumericalAlgebraicGeometry/SYSTEMS/real/elliptope2steiner.m2:4:load "NAGtools.m2"
M2/Macaulay2/packages/NumericalAlgebraicGeometry/SYSTEMS/real/nobuki.m2:3:load "NAGtools.m2"
M2/Macaulay2/packages/NumericalSchubertCalculus/EXA/1e621-G36-SLP.m2:18:needsPackage "NAGtools"
M2/Macaulay2/packages/NumericalSchubertCalculus/EXA/1e9-G36-SLP.m2:14:needsPackage "NAGtools"
M2/Macaulay2/packages/NumericalSchubertCalculus/EXA/21e3-G36-SLP.m2:11:needsPackage "NAGtools"
M2/Macaulay2/packages/NumericalSchubertCalculus/EXA/2e4-G26-SLP.m2:11:needsPackage "NAGtools"
M2/Macaulay2/packages/NumericalSchubertCalculus/EXA/2e53e2-G48-SLP.m2:17:needsPackage "NAGtools"
M2/Macaulay2/packages/NumericalSchubertCalculus/EXA/4lines-SLP.m2:25:needsPackage "NAGtools"
M2/Macaulay2/packages/ReactionNetworks/monodromy-experiment.m2:4:needsPackage "NAGtools"
```
@timduff35, some of the examples can be modified to use `MonodromySolver`. Feel free to move the ones in `NumericalAlgebraicGeometry/` to `MonodromySolver/`.

**Comment by d-torrance** [comment]
Closing - fixed in 718c3dd
